picard.wdl

```diff
@@ -608,4 +610,35 @@ task SortVcf {
         dockerImage: {description: "The docker image used for this task. Changing this may result in errors which the developers may choose not to address.",
                       category: "advanced"}
     }
-}
\ No newline at end of file
+}
+
+task RenameSample {
+    input {
+        String dockerImage = "quay.io/biocontainers/picard:2.19.0--0"
+        File inputVcf
+        String outputPath
+        String newSampleName
+        Int memory = 8
+        Float memoryMultiplier = 3.0
```

Comment:
We don't use the memory multiplier anymore. Instead, provide two memory inputs:
- memory (which is given to runtime)
- javaXmx (which is given to java)